File: Cpp/TREE/Inorder Traversal.cpp
//https://www.geeksforgeeks.org/problems/inorder-traversal/1
class Solution {
  public:
    // Function to return a list containing the inorder traversal of the tree.
    vector<int>vec;
    void inor(Node *root){
        if(root==NULL){
            return;
        }
        inor(root->left);
        vec.push_back(root->data);
        inor(root->right);
    }
    vector<int> inOrder(Node* root) {
        // Your code here
        inor(root);
        return vec; 
    }
};

File: Python/TREE/Inorder Traversal.py
//https://www.geeksforgeeks.org/problems/inorder-traversal/1
class Solution:
    def inOrder(self, root):
        result = []

        def inorderTraversal(node):
            if node is None:
                return
            inorderTraversal(node.left)
            result.append(node.data)
            inorderTraversal(node.right)

        inorderTraversal(root)
        return result
